yb-voyager/cmd/import.go

```diff
@@ -212,10 +213,9 @@ func registerImportDataFlags(cmd *cobra.Command) {
 	cmd.Flags().IntVar(&tconf.Parallelism, "parallel-jobs", -1,
 		"number of parallel copy command jobs to target database. "+defaultParallelismMsg)
 	BoolVar(cmd.Flags(), &tconf.EnableUpsert, "enable-upsert", true,
-		"true - to enable UPSERT mode on target tables\n"+
-			"false - to disable UPSERT mode on target tables")
+		"Enable UPSERT mode on target tables")
 	BoolVar(cmd.Flags(), &tconf.UsePublicIP, "use-public-ip", false,
-		"true - to use the public IPs of the nodes to distribute --parallel-jobs uniformly for data import (default false)\n"+
+		"Use the public IPs of the nodes to distribute --parallel-jobs uniformly for data import (default false)\n"+
 			"Note: you might need to configure database to have public_ip available by setting server-broadcast-addresses.\n"+
 			"Refer: https://docs.yugabyte.com/latest/reference/configuration/yb-tserver/#server-broadcast-addresses")
```

Comment:
Can you pls update this docs link to - 
Refer: https://docs.yugabyte.com/preview/reference/configuration/yb-tserver/#server-broadcast-addresses

Reply:
oh okay, then it's fine. I didn't check that, I thought it was a broken link as Aishwarya shared this change when she was looking at the help msg.